articles/ai-services/computer-vision/includes/quickstarts-sdk/identity-python-sdk.md

```diff
@@ -95,10 +98,6 @@ If you want to clean up and remove an Azure AI services subscription, you can de
 * [Portal](../../../multi-service-resource.md?pivots=azportal#clean-up-resources)
 * [Azure CLI](../../../multi-service-resource.md?pivots=azcli#clean-up-resources)
 
-To delete the **PersonGroup** you created in this quickstart, run the following code in your script:
-
-[!code-python[](~/cognitive-services-quickstart-code/python/Face/FaceQuickstart.py?name=snippet_deletegroup)]
```

Comment:
Why are we removing these instructions?

Reply:
PersonGroup deletion is included in the new sample code (FaceQuickstart-single.py) now. The PersonGroup will be automatically deleted, so I think it is fine to remove these instructions in `Clean up resources` section. 
![image](https://github.com/MicrosoftDocs/azure-docs/assets/100277363/0fb90065-48ef-464f-b43c-fce27902ab15)